internal/oci/uvm_test.go

```diff
@@ -0,0 +1,76 @@
+package oci
+
+import (
+	"testing"
+
+	runhcsopts "github.com/Microsoft/hcsshim/cmd/containerd-shim-runhcs-v1/options"
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func Test_Spec_NO_Update_MemorySize(t *testing.T) {
+
+	opts := &runhcsopts.Options{
+		VmMemorySizeInMb: 3072,
+	}
+	s := &specs.Spec{
+		Linux: &specs.Linux{},
+		Annotations: map[string]string{
+			annotationMemorySizeInMB: "2048",
+		},
+	}
+	updatedSpec := UpdateSpecFromOptions(*s, opts)
+
+	if updatedSpec.Annotations[annotationMemorySizeInMB] == "3072" {
```

Comment:
I would prefer we write this as `!= "2048"` since we know that is the expected case. Any other value should fail. Here we could return any value other than `"3072"` and pass

Reply:
Yeah, I forgot to update this. I did use != for other tests. Will update this